client/styles/register.scss

```diff
@@ -1,6 +1,107 @@
-.register-page {
-  .box {
-    margin: 0 auto;
-    max-width: 25rem;
-  }
+//.register-page {}
+.box {
+  margin: auto;
+  position: absolute;
+  width: 335px;
+  height: 700px;
+  left: 20px;
+  right: 20px;
+  top: 104px;
+  background: #ffffff;
+  border-radius: 30px 30px 0px 0px;
 }
+
+.register {
+  background: #ffffff;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.inputStyle {
+  height: 40px;
+  width: 295px;
+  border-radius: 5px;
+  border: 1px solid #000000;
+  font-family: "Heebo", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  background: #ffffff;
+  line-height: normal;
+}
+
+.userNameLabel Label {
+  height: 21px;
+  width: 295px;
+  top: 20px;
+  margin-bottom: 6px;
+  font-family: "Heebo", sans-serif;
+}
+
+.phoneLabel Label {
+  height: 21px;
+  width: 295px;
+  margin-top: 10px;
+  margin-bottom: 6px;
+  font-family: "Heebo", sans-serif;
+}
+
+.ageLabel Label {
+  height: 21px;
+  width: 295px;
+  margin-top: 10px;
+  margin-bottom: 6px;
+  font-family: "Heebo", sans-serif;
+}
+
+.formCheck {
+  width: 24px;
+  height: 24px;
+  margin-top: 30px;
+}
+
+.spantxt {
+  font-size: 16px;
+  margin-right: 10px;
+  margin-left: 25px;
+  margin-top: 30px;
+  position: absolute;
+  width: 256px;
+  height: 24px;
+  font-family: "Heebo";
+  font-style: normal;
+  font-weight: normal;
+  line-height: 23px;
+  /* identical to box height */
+  text-align: right;
+  color: #000000;
+}
+
+.button {
```

Comment:
This will override the styles of all buttons in the system. so just change the styles of the buttons that are inside the page.